### PR TITLE
Update index.md with fix

### PR DESCRIPTION
**Pull request by yogesh-io**
Replace Github Emojis with their image equivalents

### CODE REVIEW

**Comment by yogesh-io** [comment]
@googlebot I signed it!

File: docs/index.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 > Rendertron is a headless Chrome rendering solution designed to render & serialise web pages on the fly.
 
-- :hammer: Built with [Puppeteer](https://github.com/GoogleChrome/puppeteer)
-- :cloud: Easy deployment to Google Cloud
-- :mag: Improves SEO
+- ![](https://github.githubassets.com/images/icons/emoji/unicode/1f528.png) Built with [Puppeteer](https://github.com/GoogleChrome/puppeteer)
+- ![]https://github.githubassets.com/images/icons/emoji/unicode/2601.png?v8 Easy deployment to Google Cloud
+- ![]https://github.githubassets.com/images/icons/emoji/unicode/1f50d.png?v8 Improves SEO
```
Comment (AVGP) [inline docs/index.md]: The second and third emoji are missing the parentheses around the URL. The first one has them, though :)

**Comment by AVGP** [comment]
Thanks, @yogesh-io, for submitting your PR - once the syntax glitch is fixed, I'll get this merged swiftly!

Cheers! 👋

File: docs/index.md (hunk 1 above)
Comment (yogesh-io) [reply docs/index.md]: Fixed the missing parentheses error. ✌️

**Pull request merged**